test/com/xilinx/rapidwright/edif/TestEDIFNetlist.java

```diff
@@ -0,0 +1,48 @@
+package com.xilinx.rapidwright.edif;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.xilinx.rapidwright.design.Design;
+import com.xilinx.rapidwright.device.Part;
+import com.xilinx.rapidwright.device.PartNameTools;
+import com.xilinx.rapidwright.support.CheckOpenFiles;
+
+class TestEDIFNetlist {
+
+    private static final String PART_NAME = "xcvu440-flga2892-2-e";
+    
+    private static final String TEST_MACRO = "IOBUFDS_INTERMDISABLE";
+    
+    private Design createSampleMacroDesign(String macro, Part part) {
+        String designName = TEST_MACRO +"_design";
+        final EDIFNetlist netlist = EDIFTools.createNewNetlist(designName);
+        final Design design = new Design(designName, PART_NAME);
+        design.setNetlist(netlist);
+        design.setPartName(part.getName());
```

Comment:
Is this necessary give you create `Design` with PART_NAME?

Reply:
Nope